sdk/python/tests/test_bigquery_ingestion.py

```diff
@@ -0,0 +1,76 @@
+import time
+from datetime import datetime, timedelta
+
+import pandas as pd
+import pytest
+from google.cloud import bigquery
+
+from feast.data_source import BigQuerySource
+from feast.feature import Feature
+from feast.feature_store import FeatureStore
+from feast.feature_view import FeatureView
+from feast.protos.feast.types.EntityKey_pb2 import EntityKey as EntityKeyProto
+from feast.protos.feast.types.Value_pb2 import Value as ValueProto
+from feast.repo_config import LocalOnlineStoreConfig, OnlineStoreConfig, RepoConfig
+from feast.value_type import ValueType
+
+
+@pytest.mark.integration
+def test_bigquery_ingestion():
```

Comment:
We should probably also have a more representative ingestion, something like 10M rows.

Reply:
agreed. Haven't tried it but I imagine this won't go over well without paging so I'm going to defer this

Reply:
Can we perhaps just add the test and optimize it later? I believe @oavdeev will be looking at optimizing it a bit in the next week.